.github/workflows/build-nightly-release-workflow.yml

```diff
@@ -230,6 +230,5 @@ jobs:
     with:
       ice_version: ${{ needs.set-version.outputs.semver_version }}
       repository_url: "${{ inputs.repository_url }}/nightly/"
-      repository_username: ${{ inputs.repository_username }}
     secrets:
-      REPOSITORY_PASSWORD: ${{ secrets.NEXUS_NIGHTLY_PASSWORD }}
+      ICE_SWIFT_NIGHTLY_TOKEN: ${{ secrets.ICE_SWIFT_NIGHTLY_TOKEN }}
```

Comment:
Maybe we should have a shared token that we can use for the brew tap as well: `secrets. GITHUB_NIGHTLY_PAT `, and use `GITHUB_NIGHTLY_PAT` for the env name.

Reply:
What does PAT stand for?

Reply:
Personal access token.

Reply:
Created a shared token ICE_NIGHTLY_PUBLISH_TOKEN and updated the workflow to use it.